### ISSUE TITLE
option to not install system services and launch hooks on osx

### ISSUE THREAD

**Issue by rryan** [keybase/client#5829]
I'm excited to try the desktop app! I was deterred from trying it because of this user account:
https://news.ycombinator.com/item?id=13604487

Here's my situation:
* I already have keybase installed via Homebrew. That's my preferred method of installing software and keeping it up to date.
* I am not allowed to change /usr/local due to my system administrator.
* I do not want Keybase to start automatically, or install services on my computer.
* I do not want another menubar item -- it's already cluttered enough. I can open Keybase myself when I want to use it.

Is there a way to install the desktop app without it embedding itself on my computer? I can build from source.

**Comment by cjb** [comment]
Hi!  No, I'm afraid we don't support that use case.  (Also, major parts of Keybase -- like the filesystem and chat -- depend on running in the background and responding to new events.)

**Comment by rryan** [comment]
Fair enough -- does that also mean there's no option to install without giving you root?

**Comment by cjb** [comment]
You could compile from source to get that, but wouldn't be able to use the filesystem -- it's not a use case we support for that reason.
